src/java/org/apache/cassandra/db/compaction/CompactionController.java

```diff
@@ -17,10 +17,12 @@
  */
 package org.apache.cassandra.db.compaction;
 
+import java.io.Closeable;
```

Comment:
Unused import?

Reply:
ah nice thanks, forgot there's no spotless in this repo